readme.md

```diff
@@ -11,15 +11,31 @@ The user documentation for QHAna can be viewed on [qhana.readthedocs.io](https:/
 
 If you want to use the [NISQ-Analyzer](https://github.com/UST-QuAntiL/nisq-analyzer) plugin, use `docker compose --profile nisq up`.
 
+### Docker Compose V1 deprecation
+
+Docker Compose V1 and the `docker-compose` command will no longer be supported by the end of June 2023.
+`docker compose` (space instead of hyphen) should be used instead.
+For more information about the deprecation of Docker Compose V1 see https://www.docker.com/blog/new-docker-compose-v2-and-v1-deprecation/ .
```

Comment:
While many markdown tools automatically detect URLs in plain text, the markdown syntax for a link without text is `<example.com>`.

```suggestion
For more information about the deprecation of Docker Compose V1 see <https://www.docker.com/blog/new-docker-compose-v2-and-v1-deprecation/>.
```